### DISCUSSION TITLE
Help with Papercall logo

### DISCUSSION THREAD

**Discussion by colindean** [simple-icons/simple-icons#8263] [Help with SVGs] [answered]
I'm working on a submission for [Papercall](https://papercall.io), but I'm kinda stumped and hitting the limits of my timebox on this distraction. This is one of those logos that relies a good bit on color differences for a faux 3D effect. I'm taking a stab at this barely knowing what I'm doing!

I've got what I've got so far in [this branch](https://github.com/colindean/simple-icons/blob/51f20c4efe71287b742260d2cd578998626e5acf/icons/papercall.svg?short_path=4ffef32). I started from [this](https://www.papercall.io/assets/logo-papercall-e99c414d1272438ef923831124b73dfd7fa38ed07037209ebd75965e7e258954.svg). I'm using Inkscape v1.2.

Problems:

* When I try to combine paths (`Path > Combine` or `Path > Union`), I lose fidelity on the engine nacelles and flame under the plane.
* I'm not sure how best to remove all of the fill settings other than manually outside of Inkscape.
* Should the `id` attributes get cleaned up, too?

**Comment by adamrusted** [comment, marked as answer]
Hey @colindean - apologies for not getting back to you, our discussions here haven't been particularly active!
Our approach when adding these more complex icons is to group all elements in their default colour, shrink the group down so the longest dimension is 24px, centre the group, and ungroup. From there, and starting at the 'bottom' of the layer tree, we take a bit of artistic liberty by comparing the bottom-most layer, and the layer directly above it, adding a 0.5pt outline to the 'higher' layer, expanding that stroke, and using `Path > Difference` to remove from the 'lower' layer. Combine the two remaining paths to make the 'flat' logo. Repeat this for every layer in the SVG - and you should have yourself a single path 'simple' icon!
